doc/howtos/firewall-configuration.rst

```diff
@@ -119,3 +119,11 @@ Opening holes to firewall using this method is not absolutely mandatory.
 If you are building a device without dynamic services or applications,
 you can simply add the necessary rules to the custom ruleset that is
 loaded upon device startup.
+
+Note that if you are starting your service as non-root user (as you
+probably should be) using systemd's ``User=`` option, you'll need to
+make sure that the firewall manipulation commands are run with original
+permissions. An easy way to do that is to use
+``PermissionsStartOnly=true`` option in the service file. This works if
+you don't need to run any other ``ExecStartPre=`` or similar commands
+with the reduced non-root privileges.
```

Comment:
Where would you put such line? In the service file itself or the *.conf file? I'm asking because I'm also wondering how that works in the case your service file is auto-generated by the AppFW.